src/unix/bsd/freebsdlike/freebsd/x86_64/mod.rs

```diff
@@ -78,6 +80,16 @@ s_no_extra_traits! {
         pub xmm_reg: [[u8; 16]; 8],
         pub xmm_pad: [u8; 224],
     }
+
+    #[cfg_attr(feature = "extra_traits", derive(Debug))]
```

Comment:
This is unnecessary since `s!` macro already derives it.

Reply:
No, this is `s_no_extra_traits`!

I guess since we implement everything on `mcontext`, `ucontext` could go into `s`..